docs/index.rst

```diff
@@ -1,60 +1,81 @@
   </embed>
 
-*Modin is a library for unifying the way you interact with your data*
+.. code-block:: python
+
+  # import pandas as pd
+  import modin.pandas as pd
+
 
-Modin can be installed with pip: ``pip install modin``
+Scale your pandas workflow by changing a single line of code.
+-------------------------------------------------------------
+
+Modin uses Ray_ to provide an effortless way to speed up your pandas notebooks, scripts,
+and libraries. Unlike other distributed DataFrame libraries, Modin provides seamless
+integration and compatibility with existing pandas code. Even using the DataFrame
+constructor is identical.
+
+.. code-block:: python
 
-Pandas on Ray
--------------
+  import modin.pandas as pd
+  import numpy as np
 
-*Pandas on Ray is a project designed to effortlessly scale your pandas code requiring a change of only a single line of code*
+  frame_data = np.random.randint(0, 100, size=(2**10, 2**8))
+  df = pd.DataFrame(frame_data)
 
-+----------------------------------------------+-------------------------------------------------+
-| **pandas**                                   | **Pandas on Ray**                               |
-+----------------------------------------------+-------------------------------------------------+
-|.. code-block:: python                        |.. code-block:: python                           |
-|                                              |                                                 |
-| # Normal pandas import                       | # Pandas on Ray import                          |
-| import pandas as pd                          | import modin.pandas as pd                       |
-|                                              |                                                 |
-| df = pd.DataFrame({'col1': [1, 2, 3],        | df = pd.DataFrame({'col1': [1, 2, 3],           |
-|                    'col2': [1.0, 2.0, 3.0]}) |                    'col2': [1.0, 2.0, 3.0]})    |
-|                                              |                                                 |
-| df.sum()                                     | df.sum()                                        |
-| ...                                          | ...                                             |
-+----------------------------------------------+-------------------------------------------------+
+To use Modin, you do not need to know how many cores your system has and you do not need
+to  specify how to distribute the data. In fact, you can continue using your previous
+pandas notebooks while experiencing a considerable speedup from Modin, even on a single
```

Comment:
Minor idea, but maybe state how the API is exactly the same is pandas in this intro section (as well as what is already there in later sections), just to make it pop out more for someone reading for the first time. It's sort of clear right now, but having it explicitly said here might make it better.

Reply:
Added, thanks!